catalog/core/catalog-core-api/src/main/java/ddf/catalog/event/Subscription.java

```diff
@@ -36,4 +36,18 @@
      */
     public DeliveryMethod getDeliveryMethod();
 
+    /**
```

Comment:
I agree that giving implementation direction seems odd here.

Also do we really need both methods?  Seems like you could just have one `isAvailable` and it would be an implementation detail for it to cache the ping response.